srcpkgs/mantablockscreen/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'mantablockscreen'
+pkgname=mantablockscreen
+version=0.3
+revision=1
+conflicts="i3lock"
+depends="ImageMagick xrandr i3lock-color compton"
+short_desc="Another i3lock-color wrapper that mantab"
+maintainer="Leonardo F. <leonardof@tutanota.com>"
+license="MIT"
+homepage="https://github.com/reorr/mantablockscreen"
+distfiles="https://github.com/reorr/mantablockscreen/archive/v${version}.tar.gz>mantablockscreen.tar.gz"
+checksum=796a5a16ef6392f2addb6a9bf1412b38a02aea7ee317d2ba8fce5fce08f6dfc0
+
+do_install() {
+    vbin mantablockscreen
```

Comment:
the makefile also installs to /usr/share/mantabassets/ so it needs.

```
vmkdir usr/share/mantabassets
vcopy mantabassets/* usr/share/mantabassets
```

as well